test/gateways/accountHolderApiGateway.test.ts

```diff
@@ -34,21 +31,21 @@ describe("Account Holder API Gateway", () => {
 
       await gateway.getAccountHolderId(authId, token);
 
-      expect((axios as any).get).toHaveBeenLastCalledWith(expectedUrl, {
+      expect(mockedAxios.get).toHaveBeenLastCalledWith(expectedUrl, {
         headers: { Authorization: `Bearer ${token}` },
       });
     });
 
     it("should return the obtained accountHolderId from the API", async () => {
       mockedAxios.get.mockResolvedValue({
         data: {
-          id: acountHolderId,
+          id: accountHolderId,
```

Comment:
Where is `accountHolderId` set 🤔

Reply:
yes you're right, this is a refactor hangover, have shuffled around the `let`s and the `const`s to make clearer 👍